pkg/ibmvpc/examples/acl_testing3_with_grouping.txt

```diff
@@ -0,0 +1,10 @@
+
+grouped output:
+vsi1-ky[10.240.10.4] => 161.26.0.0/16 : protocol: UDP  
+vsi1-ky[10.240.10.4] => vsi2-ky[10.240.20.4] : protocol: TCP,UDP  
+vsi2-ky[10.240.20.4] => 142.0.0.0/8 : protocol: ICMP  
+vsi2-ky[10.240.20.4] => vsi1-ky[10.240.10.4] : All Connections
+vsi3a-ky[10.240.30.5],vsi3b-ky[10.240.30.6] => vsi3c-ky[10.240.30.4] : All Connections
+vsi3a-ky[10.240.30.5],vsi3b-ky[10.240.30.6],vsi3c-ky[10.240.30.4] => vsi1-ky[10.240.10.4] : All Connections
+vsi3a-ky[10.240.30.5],vsi3c-ky[10.240.30.4] => vsi3b-ky[10.240.30.6] : All Connections
+vsi3b-ky[10.240.30.6],vsi3c-ky[10.240.30.4] => vsi3a-ky[10.240.30.5] : All Connections
```

Comment:
```suggestion
vsi3a-ky[10.240.30.5],vsi3b-ky[10.240.30.6],vsi3c-ky[10.240.30.4] => vsi3a-ky[10.240.30.5],vsi3b-ky[10.240.30.6],vsi3c-ky[10.240.30.4] : All Connections
vsi3a-ky[10.240.30.5],vsi3b-ky[10.240.30.6],vsi3c-ky[10.240.30.4] => vsi1-ky[10.240.10.4] : All Connections
```
This requires adding a self loop to each VSI, run the grouping algorithm, then remove left-over self loops.
Can be handled in a future PR.

Reply:
opened new issue